continuous_integration/test_script.sh

```diff
@@ -8,6 +8,10 @@
 
 set -e
 
+cd /tmp
+mkdir sklearn_temp
+cd sklearn_temp
```

Comment:
We can thus check that running test in a directory does not leave artefacts at the end